astropy/visualization/stretch.py

```diff
@@ -299,16 +304,20 @@ class InvertedLogStretch(BaseStretch):
     The stretch is given by:
 
     .. math::
-        y = \frac{e^{y} (a + 1) -1}{a}
+        y = \frac{e^{y \log{a + 1}} - 1}{a} \\
+        y = \frac{e^{y} (a + 1) - 1}{a}
 
     Parameters
     ----------
     a : float, optional
-        The ``a`` parameter used in the above formula.  Default is 1000.
+        The ``a`` parameter used in the above formula.  ``a`` must be >
+        0.  Default is 1000.
```

Comment:
I think it would be better to write out 'must be > 0' in words, and similar comment in other docstrings (I found it quite hard to read with the line break)

Reply:
Done.